stata_kernel/code_manager.py

```diff
@@ -218,22 +218,33 @@ def get_text(self, config):
             use_include = True
 
         # Insert `graph export`
-        graph_fmt = config.get('graph_format')
-        graph_scale = config.get('graph_scale')
+        over_fmt = config.overrides['plot']['format']
+        over_scale = config.overrides['plot']['scale']
+        over_width = config.overrides['plot']['width']
+
+        graph_fmt = over_fmt if over_fmt else config.get('graph_format')
+        graph_scale = over_scale if over_scale else config.get('graph_scale')
+        graph_width = over_width if over_width else 600
+        graph_height = config.overrides['plot']['height']
```

Comment:
So these settings allow for manual aspect ratio sizing, right? I assume there's no issue if a user hasn't set a default in their `conf` file?

Reply:
So `config.overrides` will give what if it doesn't exist?

Reply:
The idea was to set it locally or to whatever is in `config`. I'll change it to work with `env`

Reply:
:+1: I've been meaning to try to figure out how to write the configuration to file including comments, but I haven't done that yet. So as of now, writing the configuration to file deletes the initial explanatory comments written during the initial `install.py` script.